service/observiq-otel-collector.service

```diff
@@ -17,5 +17,6 @@ TimeoutSec=120
 StandardOutput=journal
 Restart=on-failure
 RestartSec=5s
+KillMode=process
```

Comment:
It will not orphan the JMX I don't think based on the way OTel kicks it off and shuts it down. It might orphan it if we need to SIGKILL the collector. I can try `mixed` mode which sends a SIGINT to the main process but SIGKILL to all child processes.

Reply:
`KillMode` has to be `process` looks like. It won't orphan the updater otherwise. Looking at the [code](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/jmxreceiver/internal/subprocess/subprocess.go#L134-L155) for how the JMX jar is shutdown it can already be zombied during normal shutdown if it takes to long. I think this is probably an unlikely case that we shouldn't optimize for right now.